### ISSUE TITLE
EvaluateFewShot in MAML

### ISSUE THREAD

**Issue by thanhhau097** [oscarknagg/few-shot#32]
In the code, the model is passed to Evaluation phase by using set_model() function:

```python
    def set_model(self, model):
        for callback in self.callbacks:
            callback.set_model(model)
````

Does it work correctly if we only copy the weights/state_dict of the model and optimizer? (In the case that we save the model weight and then finetune with new tasks) Is there any loss information when we save the model state_dict?
